src/app/components/toolbar/toolbar-guidelines.component.html

```diff
@@ -48,13 +42,12 @@
 </section>
 
 <section>
-  <header><div class="sky-section-heading">Layout</div></header>
-  <div class="sky-subsection-heading">Anatomy of a toolbar</div>
+	<header><div class="sky-section-heading">Composition</div></header>
   <div class="screenshot screenshot-info"><img src="~/assets/img/guidelines/toolbar/anatomy.png" style="width: 100%;" /></div>
 
-  <div class="sky-subsection-heading">Action button order</div>
+  <div class="sky-subsection-heading">Standard toolbar actions</div>
   <p>
-    Toolbar buttons should always appear in a consistent order. Not all toolbars will have every possible button, but when a button is present, it should be in the expected location. Standard actions should appear from left to right in the following order when present in a toolbar.
+    Not all toolbars will have every possible button, but when a button is present, it should be in the expected location. Standard actions should appear from left to right in the following order when present in a toolbar.
```

Comment:
I know this is text that was already in place, but I'd tweak the opening sentence to: "Toolbars don't always have every possible button, but when present, buttons should be in the expected locations." And I'd change the period at the end of the second sentence to a colon.

Reply:
ok